configs/rule_bindings/team-4-rule-bindings.yml

```diff
@@ -0,0 +1,71 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+rule_bindings:
+  T4_URI_DP_EMAIL_DUPLICATE:
```

Comment:
there should be a test_templates generated sql test for each of these rules

Reply:
check that with each of the rule bindings, we can generate the same expected intermediate json
have one additional test that creates a sql from one of the rule_bindings

Reply:
todo: amandeep: add test_templates to test intermediate json and output sql